package.json

```diff
@@ -35,5 +35,8 @@
     "ts-jest": "^27.1.0",
     "typescript": "^4.2.4"
   },
+  "resolutions": {
+    "fs-extra": "^9.0.0"
```

Comment:
See description for the reason for this

Reply:
`fs-extra@10.1.0` has been released to address https://github.com/jprichardson/node-fs-extra/pull/953.

Reply:
`fs-extra@10.1.0` has been released to address https://github.com/jprichardson/node-fs-extra/pull/953.
Might want to bump it up instead, maybe?